src/main/java/com/spring/task/gymcrm/dto/LoginRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.spring.task.gymcrm.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class LoginRequest {
+    @NotNull(message = "Username required!")
+    private String username;
```

Comment:
It's time to move all constants to separate class.

Reply:
You mean messages here?

Reply:
Correct.